llvm/include/llvm/CodeGen/InterferenceCache.h

```diff
@@ -142,7 +142,7 @@ class LLVM_LIBRARY_VISIBILITY InterferenceCache {
 
   // Point to an entry for each physreg. The entry pointed to may not be up to
   // date, and it may have been reused for a different physreg.
-  unsigned char* PhysRegEntries = nullptr;
+  unsigned char *PhysRegEntries = nullptr;
```

Comment:
Reformat shouldn't be done at same time as the move

Reply:
Yes I can put the new greedy pass in a header instead.